test/utils/index.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs')
+
+/**
+ *
+ * @param {string} pathFile
+ * @param {number} now Date.now()
+ */
+function insertDateOnFile (pathFile, now) {
```

Comment:
@luizAlbuquerqueJr, só questão de inglês: o nome correto pra o que a função faz seria: insertTimestamp**In**File

Reply:
Done